app-config.yaml

```diff
@@ -10,6 +10,13 @@ backend:
     methods: [GET, POST, PUT, DELETE]
     credentials: true
 
+proxy:
+  '/circleci/api':
```

Comment:
Internally we put the api keyword before like 'api/circleci' is proxied to 'https://circleci.com/api/v1.1'

should we stick to that?

Reply:
It's prefixed with `/proxy` atm, so this would be `/proxy/circleci/api`. I'm thinking we could use `/proxy` for these proxy configurations and possibly `/api` for backend plugins.